opensquirrel/utils/__init__.py

```diff
@@ -1,6 +1,18 @@
+from opensquirrel.utils.check_mapper import check_mapper
 from opensquirrel.utils.identity_filter import filter_out_identities
 from opensquirrel.utils.list import flatten_list
 from opensquirrel.utils.math import acos, are_axes_consecutive
-from opensquirrel.utils.matrix_expander import get_matrix
+from opensquirrel.utils.matrix_expander import can1, expand_ket, get_matrix, get_reduced_ket
 
-__all__ = ["acos", "are_axes_consecutive", "filter_out_identities", "flatten_list", "get_matrix"]
+__all__ = [
+    "acos",
+    "are_axes_consecutive",
+    "filter_out_identities",
+    "flatten_list",
+    "get_matrix",
+    "check_mapper",
+    "expand_ket",
+    "get_reduced_ket",
+    "can1",
+    "get_matrix",
+]
```

Comment:
Nice! I think, even these identifiers being internal, we want to have them in this init file for our own sake (to make their referencing easier from everywhere).

Reply:
Have we checked also that, everywhere in the code, we just do `from opensquirrel.utils import...` and not `from opensquirrel.utils.blah import...`?

Reply:
I've checked it for every import that I added to init. I'll double check just to be sure.